examples/typescript-decorators/README.md

```diff
@@ -1,7 +1,7 @@
-# TypeScript decorators example using `express-decorators` package
-
-Minimal [TypeScript](https://www.typescriptlang.org/) setup for tinyhttp using [ts-node](https://github.com/TypeStrong/ts-node).
+# TypeScript decorators example
 
+Minimal [TypeScript](https://www.typescriptlang.org/) example that uses [ts-node](https://github.com/TypeStrong/ts-node) and [express-decorators](https://github.com/sjmeverett/express-decorators) package in order to organize the HTTP methods in controller classes.
+This way you can separate the logic based on the entities you use.
```

Comment:
u forgot to put a space before heading :D

prob prettier didn't fix the indentation